app-dicts/myspell-it/Manifest

```diff
@@ -1 +1,2 @@
 DIST Dizionari.IT_20081129.oxt 1433036 BLAKE2B 0e0aa03fafa34957a00b054070de67aec3f742d4cf177c6185c8a6b8a4005ea94f4041fb2e29a0ab7fa87a7d04c5f2148f98bd446e196470104ccf1b3ad97a0c SHA512 f0576f62448d368680545a66bf99d3d44c75562a0b5bc72c9020cef707c5543182ea1f45934826970b9e3ea588b9f2e29686cd08f7b8fdddcfdbd067e6536071
+DIST myspell-it-20150925.oxt 1373512 BLAKE2B 68bc80e758bd28edba5251af83c3e372ffde87335a4844c1e3a05942449db385a950d91f7a07db02393dd79badeaedd9465a8b2812ace9299e9e73e4fb76ffc7 SHA512 5b251aeaba2bb5548cb350e670595ed04e87b0fcbbb77789ad17f3aa8ff621552e1793e94919d088af35f25d0e7f8f0f9695ad298d346914609848907c9b7404
```

Comment:
```
>>> Emerging (1 of 1) app-dicts/myspell-it-20150925::gentoo
>>> Failed to emerge app-dicts/myspell-it-20150925, Log file:
>>>  '/var/tmp/portage/vbslogs/build/app-dicts/myspell-it-20150925:20210603-101240.log'
>>> Jobs: 0 of 1 complete, 1 failed                 Load avg: 0.78, 0.41, 0.17
>>> Downloading 'https://ftp-stud.hs-esslingen.de/pub/Mirrors/gentoo/distfiles/6b/myspell-it-20150925.oxt'
--2021-06-03 10:12:40--  https://ftp-stud.hs-esslingen.de/pub/Mirrors/gentoo/distfiles/6b/myspell-it-20150925.oxt
Resolving ftp-stud.hs-esslingen.de... 129.143.116.10, 2001:7c0:700::10
Connecting to ftp-stud.hs-esslingen.de|129.143.116.10|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2021-06-03 10:12:40 ERROR 404: Not Found.

>>> Downloading 'https://extensions.libreoffice.org/assets/downloads/z/874d181c_dict-it.oxt'
--2021-06-03 10:12:40--  https://extensions.libreoffice.org/assets/downloads/z/874d181c_dict-it.oxt
Resolving extensions.libreoffice.org... 89.238.68.135, 2a00:1828:a012:135::1
Connecting to extensions.libreoffice.org|89.238.68.135|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 1373663 (1.3M) [application/vnd.openofficeorg.extension]
Saving to: '/var/cache/distfiles/myspell-it-20150925.oxt.__download__'

     0K .......... .......... .......... .......... ..........  3%  200K 6s
    50K .......... .......... .......... .......... ..........  7%  316K 5s
   100K .......... .......... .......... .......... .......... 11%  554K 4s
   150K .......... .......... .......... .......... .......... 14%  497K 3s
   200K .......... .......... .......... .......... .......... 18%  539K 3s
   250K .......... .......... .......... .......... .......... 22%  804K 3s
   300K .......... .......... .......... .......... .......... 26%  601K 2s
   350K .......... .......... .......... .......... .......... 29%  413K 2s
   400K .......... .......... .......... .......... .......... 33%  674K 2s
   450K .......... .......... .......... .......... .......... 37%  467K 2s
   500K .......... .......... .......... .......... .......... 40%  522K 2s
   550K .......... .......... .......... .......... .......... 44%  532K 2s
   600K .......... .......... .......... .......... .......... 48%  331K 2s
   650K .......... .......... .......... .......... .......... 52%  542K 1s
   700K .......... .......... .......... .......... .......... 55%  709K 1s
   750K .......... .......... .......... .......... .......... 59%  950K 1s
   800K .......... .......... .......... .......... .......... 63% 1.02M 1s
   850K .......... .......... .......... .......... .......... 67% 1.17M 1s
   900K .......... .......... .......... .......... .......... 70% 2.28M 1s
   950K .......... .......... .......... .......... .......... 74% 1.43M 1s
  1000K .......... .......... .......... .......... .......... 78% 1.33M 1s
  1050K .......... .......... .......... .......... .......... 81% 1.10M 0s
  1100K .......... .......... .......... .......... .......... 85% 2.50M 0s
  1150K .......... .......... .......... .......... .......... 89%  552K 0s
  1200K .......... .......... .......... .......... .......... 93%  163M 0s
  1250K .......... .......... .......... .......... .......... 96%  945K 0s
  1300K .......... .......... .......... .......... .         100% 5.50M=2.1s

2021-06-03 10:12:43 (641 KB/s) - '/var/cache/distfiles/myspell-it-20150925.oxt.__download__' saved [1373663/1373663]

!!! Fetched file: myspell-it-20150925.oxt VERIFY FAILED!
!!! Reason: Filesize does not match recorded size
!!! Got:      1373663
!!! Expected: 1373512
Refetching... File renamed to '/var/cache/distfiles/myspell-it-20150925.oxt._checksum_failure_.psvyfgd0'

 * Fetch failed for 'app-dicts/myspell-it-20150925', Log file:
 *  '/var/tmp/portage/vbslogs/build/app-dicts/myspell-it-20150925:20210603-101240.log'
!!! Couldn't download 'myspell-it-20150925.oxt'. Aborting.
```
:o

Reply:
Fixed, I don't know why but worked on mine. I have removed and remade the Manifest, now it should be ok